task_processing/plugins/mesos/constraints.py

```diff
@@ -0,0 +1,75 @@
+import re
+
+from pyrsistent import field
+from pyrsistent import PRecord
+
+
+def equals_op(expected_value, actual_value):
+    return expected_value == actual_value
+
+
+def notequals_op(expected_value, actual_value):
+    return expected_value != actual_value
+
+
+def like_op(like_pattern, actual_value):
+    escaped_pattern = re.escape(like_pattern)
+    re_pattern = escaped_pattern.replace('\\%', '.*').replace('_', '.')
```

Comment:
what is this line for?

Reply:
that's the wildcard syntax for SQL's LIKE operator

Reply:
Yeah, eventually we agreed that it's probably better to use old good regexs instead of SQL LIKE here. :)